tests/lobster/datasets/test_m3_20m_dataset.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from lobster.datasets import M320MDataset
+
+
+@pytest.fixture(autouse=True)
+def dataset() -> M320MDataset:
+    return M320MDataset()
+
+
+class TestM320MDataset:
+    """Unit tests for M320MDataset."""
+
+    def test___init___subset(self, dataset: M320MDataset):
+        item0 = dataset[0]
+        assert len(item0) == 2
+        assert isinstance(item0[0], str)
+        assert isinstance(item0[1], str)
+        assert len(item0[1]) > 50
+
+    @pytest.mark.skip(reason="Requires download.")
```

Comment:
can you add a mocked test like in https://github.com/Genentech/beignet/blob/main/tests/beignet/datasets/test__uniprot_dataset.py#L14 ?